components/profile_popover/index.js

```diff
@@ -23,10 +23,10 @@ import ProfilePopover from './profile_popover.jsx';
 
 function mapStateToProps(state, ownProps) {
     const team = getCurrentTeam(state);
-    const teamMember = getTeamMember(state, team.id, ownProps.user.id) || {};
+    const teamMember = getTeamMember(state, team.id, ownProps.userId);
```

Comment:
See comment below, but would it make sense to leverage `initialOwnProps` instead?